test/test_configuration.py

```diff
@@ -69,8 +73,10 @@ def test_ovos_core_config_init(self):
         from neon_utils.configuration_utils import \
             get_mycroft_compatible_config
         from mycroft.configuration import Configuration as MycroftConfig
+        from neon_core.util.runtime_utils import use_neon_core
+
         mycroft_config = MycroftConfig.get()
-        neon_config = get_mycroft_compatible_config()
+        neon_config = use_neon_core(get_mycroft_compatible_config)()
```

Comment:
shouldnt `get_mycroft_compatible_config` be directly decorated where it is defined? when do we not want it to use neon?

Reply:
Possibly; I left it open since in theory a `NeonSkill` running under Mycroft/OVOS might have a call to this method that shouldn't be decorated.. I also don't want to tie neon_utils methods to NeonCore in general